app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+import os
+from models import Actor
+
+
+ABSOLUTE_PATH = os.path.abspath("../cinema.sqlite")
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(ABSOLUTE_PATH)
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+
+        return [
+            Actor(*row) for row in actors_cursor
+        ]
+
+    def update(
+            self,
+            id_to_update: int,
+            new_first_name: str,
+            new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} "
+            "SET first_name = ?, last_name = ? "
+            "WHERE id = ?",
+            (new_first_name, new_last_name, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
+        self._connection.execute(
+            f"DELETE FROM {self.table_name} WHERE id = ?",
+            (id_to_delete, )
+        )
+        self._connection.commit()
```

Comment:
Similar to other operations, consider handling exceptions for the delete operation to manage any potential errors effectively.